components/member_list_channel/member_list_channel.jsx

```diff
@@ -61,29 +62,36 @@ export default class MemberListChannel extends React.PureComponent {
         this.props.actions.setModalSearchTerm('');
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
-        if (this.props.searchTerm !== nextProps.searchTerm) {
+    static getDerivedStateFromProps(nextProps, prevState) {
+        if (nextProps.searchTerm !== prevState.searchTerm) {
+            return {searchTerm: nextProps.searchTerm};
+        }
+        return null;
+    }
```

Comment:
I think we can remove this since we're not using `this.state.searchTerm` anywhere outside of this.